source/codegen/metadata/nirfmxinstr_restricted/functions.py

```diff
@@ -957,27 +894,14 @@
             },
             {
                 'direction': 'in',
-                'name': 'selectorString',
-                'type': 'char[]'
-            },
-            {
-                'direction': 'in',
-                'name': 'arraySize',
-                'type': 'int32'
-            },
-            {
-                'direction': 'out',
-                'name': 'externalAttenuationTableNames',
-                'size': {
-                    'mechanism': 'ivi-dance',
-                    'value': 'arraySize'
-                },
+                'name': 'filePath',
                 'type': 'char[]'
             }
         ],
         'returns': 'int32'
     },
-    'GetActiveTableName': {
+    'LoadConfigurationsFromJSON': {
```

Comment:
This is the one functional change in this file, adding a custom implementation for LoadConfigurationsFromJSON to infer the arraySize from the jsonString parameter.